assets/js/src/bulk-delete.js

```diff
@@ -216,6 +216,14 @@ jQuery(document).ready(function () {
 	BulkWP.validateUserRole = function(that) {
 		return (null !== jQuery(that).parent().prev().find(".enhanced-role-dropdown").val());
 	};
+
```

Comment:
Let's move these two functions to the `assets/js/src/delete-terms.js` file.

That way we can keep all delete terms related functions in a single place.

Reply:
Fixed it.